packages/dx-react-grid-bootstrap4/src/templates/table-band-header/cell.jsx

```diff
@@ -3,12 +3,15 @@ import * as PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 export const Cell = ({
-  column, children,
+  column, children, leftBorder,
```

Comment:
What about docs for the `leftBorder` prop?

Reply:
Done